controllers/localvolumeset/localvolumeset_controller.go

```diff
@@ -87,7 +88,13 @@ func (r *LocalVolumeSetReconciler) reconcile(ctx context.Context, request reconc
 	}
 
 	// store a one to many association from storageClass to LocalVolumeSet
-	r.LvSetMap.DeregisterStorageClassOwner(lvSet.Spec.StorageClassName, request.NamespacedName)
+	r.LvSetMap.RegisterStorageClassOwner(lvSet.Spec.StorageClassName, request.NamespacedName)
+
```

Comment:
is this a new bug?

Reply:
I think this bug happened with the migration PR